test/fixtures/api-client/move.find.json

```diff
@@ -59,6 +59,9 @@
       },
       "original_move": {
         "data": null
+      },
+      "timeline_events": {
+        "data": []
```

Comment:
Would it be useful to have a fixture with a fleshed out `timeline_events` structure?

Reply:
I am following the pattern as established in `documents` and `court_hearings`. And analogous to the hasOne relationships such as `supplier` using null.

Given that the tests (currently) only check for the presence of the property, I don't believe it is useful

Reply:
@solidgoldpig explained offline in a bit more detail what these `api-client` tests cover, so I'm happy with these fixtures as they are.